spec/meilisearch/client/snapshots_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+RSpec.describe 'MeiliSearch::Client - Snapshots' do
+  it 'creates a new snapshot' do
+    response = client.create_snapshot
+    expect(response).to be_a(Hash)
+    expect(response['taskUid']).to_not be_nil
+    expect(response['status']).to_not be_nil
+    expect(response['status']).to eq('enqueued')
+    response = client.wait_for_task(response['taskUid'])
+    expect(response['status']).to eq('succeeded')
```

Comment:
You don't need to check if the method was successful or not, you can remove this two lines :)

```suggestion
```